src/psyclone/psyir/backend/fortran.py

```diff
@@ -1055,16 +1055,18 @@ def filecontainer_node(self, node):
         :rtype: str
 
         :raises VisitorError: if the attached symbol table contains \
-            any data symbols.
+            any non-routine symbols.
```

Comment:
Can you remove the backslashes in this docstring?

Reply:
Done